CMake/resolve_dependency_modules/qatzstd.cmake

```diff
@@ -37,10 +37,17 @@ set(QATZSTD_INCLUDE_DIR "${QATZSTD_SOURCE_DIR}/src")
 set(QATZSTD_STATIC_LIB_TARGETS "${QATZSTD_SOURCE_DIR}/src/libqatseqprod.a")
 set(QATZSTD_MAKE_ARGS "ENABLE_USDM_DRV=1" "ZSTDLIB=${ZSTD_INCLUDE_DIR}")
 
+set(QATZSTD_SOURCE_URL
+      "https://github.com/intel-collab/applications.qat.shims.zstandard.qatzstdplugin.git")
+set(QATZSTD_SOURCE_BRANCH "main")
+
 ExternalProject_Add(
   qatzstd
-  URL ${VELOX_QATZSTD_SOURCE_URL}
-  URL_HASH ${VELOX_QATZSTD_BUILD_SHA256_CHECKSUM}
+  # URL ${VELOX_QATZSTD_SOURCE_URL}
+  # URL_HASH ${VELOX_QATZSTD_BUILD_SHA256_CHECKSUM}
+  GIT_REPOSITORY ${QATZSTD_SOURCE_URL}
+  GIT_TAG ${QATZSTD_SOURCE_BRANCH}
+  USES_TERMINAL_DOWNLOAD ON
```

Comment:
You could also use the link in L22+ and point it to an arbitrary commit (or branch). See [gh docs](https://docs.github.com/en/repositories/working-with-files/using-files/downloading-source-code-archives#source-code-archive-urls)